src/Umbraco.Infrastructure.PublishedCache/PublishedElement.cs

```diff
@@ -14,7 +14,7 @@ namespace Umbraco.Web.PublishedCache
     // an entirely new models factory + not even sure it makes sense at all since
     // sets are created manually todo yes it does! - what does this all mean?
     //
-    internal class PublishedElement : IPublishedElement
```

Comment:
This should exist in abstractions, it has no ties to NuCache and would be used by other implementations